### PR TITLE
chore: improve comment for docker compose file

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

make it clear

## Testing

/

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -12,7 +12,8 @@ services:
     environment:
       - TRUST_PROXY_HEADER=1
       - DB_URL=postgres://postgres:p0stgr3s@postgres:5432/logto
-      - ENDPOINT # Mandatory. Mapping env to the container.
+      # Mandatory. Mapping host env to the container thus GitPod can dynamically configure the pulic URL of Logto.
```
Comment (charIeszhao) [inline docker-compose.yml:15]:
```suggestion
      # Mandatory. Mapping host env to the container thus GitPod can dynamically configure the public URL of Logto.
```

Comment (charIeszhao) [reply docker-compose.yml:15]: Btw, why mention `GitPod`? ENDPOINT is not something specific for GitPod, right?

Comment (gao-sun) [reply docker-compose.yml:15]: see [this line](https://github.com/logto-io/logto/blob/master/.gitpod.yml#L17)

Comment (wangsijie) [reply docker-compose.yml]: GitPod is just an example usage?

Comment (gao-sun) [reply docker-compose.yml]: > GitPod is just an example usage?

so does `docker-compose` - i think i can mention it also fits for dev

Comment (charIeszhao) [reply docker-compose.yml]: Well, I'm OK then

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**